snap/epoch_test.go

```diff
@@ -131,6 +131,67 @@ func (s epochSuite) TestGoodEpochs(c *check.C) {
 	}
 }
 
+func (s epochSuite) TestGoodEpochsInSnapYAML(c *check.C) {
+	defer snap.MockSanitizePlugsSlots(func(snapInfo *snap.Info) {})()
+
+	type Tt struct {
+		s string
+		e snap.Epoch
+	}
+
+	tests := []Tt{
+		{s: `epoch: 0`, e: snap.Epoch{Read: []uint32{0}, Write: []uint32{0}}},
+		{s: ``, e: snap.Epoch{Read: []uint32{0}, Write: []uint32{0}}},
+		{s: `epoch: "0"`, e: snap.Epoch{Read: []uint32{0}, Write: []uint32{0}}},
+		{s: `epoch: {}`, e: snap.Epoch{Read: []uint32{0}, Write: []uint32{0}}},
```

Comment:
was it agreed that is valid syntax in snap.yaml?

Reply:
yes, i'll be pushing a different pr where I check for dupes, other than that the store is adopting all our validation checks

Reply:
I might change Unset to IsZero, also

Reply:
> I might change Unset to IsZero, also

sounds a good idea